### ISSUE TITLE
Coordinates HUD showing west when it should show south

### ISSUE THREAD

**Issue by elrac** [MC-Machinations/VanillaTweaks#26]
I noticed that the instant that I rotate from facing in the negative direction to the positive direction but still facing south, coordinates hub shows W for west. SW never shows, it only shows W until I rotate enough to see NW.
![2021-05-08_12 15 34](https://user-images.githubusercontent.com/4564119/117551000-32904d00-aff8-11eb-902a-db05fd09a2b1.png)

**Comment by ScreaminMonk** [comment]
Hope this gets looked at. Still having the same problem on our server with this. I mentioned it back in December on the Discord and never got a response.

**Comment by Machine-Maker** [comment]
I cannot replicate this. I'm on VanillaTweaks v0.1.17.
![image](https://user-images.githubusercontent.com/15055071/117553270-61f98680-b005-11eb-8a17-6f64fd805f9b.png)
Make sure you have the latest version of the plugin installed.
What server architecture are you using? I've been doing my testing with Paper, but maybe this is a spigot issue?

**Comment by ScreaminMonk** [comment]
On my test server, I just updated to v0.1.17 from 0.1.15. It's still doing it for me. Screenshots shows plugins that are being used currently as well as datapacks in another. This server is mimicing another one that I admin for where we are using some basic plugins like Coreprotect, VT, and Wandering Trades. DPacks include things like dragon proofing some blocks, one player sleep, and armorstands. I removed them here for this test. See shots for chat displaying that what is installed.
You will see in one shot that I'm facing SW at +45º on the horizontal azimuth and in the other shot, I'm almost due south and it still hasn't changed to S on the HUD. But when facing any of the other 3-45º angles it registers fine. See NW shot for example. This is exactly what is happening on the the main server too just that there the PIs and DPs are running. 
This is on Paper 1.16.5 Build #645 
Side note, I know the clock doesn't like one player sleep either as when I go into the nether after sleeping in the OW, the clocks are not aligned with each other anymore. It's like the OW will change but the Nether fails to check and keeps on it's own clock.
![1 NW](https://user-images.githubusercontent.com/53252959/117556115-14901000-b02b-11eb-9834-4962291e570c.png)
![2 SW](https://user-images.githubusercontent.com/53252959/117556117-1a85f100-b02b-11eb-88a5-17191d1665ad.png)
![3 S](https://user-images.githubusercontent.com/53252959/117556118-1d80e180-b02b-11eb-8a85-84b760e6c0a1.png)
